app/controllers/organizations_controller.rb

```diff
@@ -1,9 +1,10 @@
 class OrganizationsController < ApplicationController
   layout 'application'
-  before_action :show_side_bar, only: [:link_organizations]
+  before_action :hide_sidebar, only: [:link_organizations]
+  before_action :show_sidebar, except: [:link_organizations]
   before_action :authenticate_github_user
   before_action :save_cookie_url
-  before_action :load_organization, except: [:index, :missing, :link_organizations]
+  before_action :load_organization, except: [:index, :missing, :link_organizations, :tracked_organizations]
```

Comment:
**[rubocop]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Layout/LineLength: Line is too long. [107/100]